webserver/server/app/services/TransactionHistoryService.ts

```diff
@@ -76,7 +82,99 @@ export async function historyForAddresses(
       transaction: {
         hash: entry.hash.toString('hex'),
         payload: entry.payload.toString('hex'),
+        outputs: computeOutputs(entry.payload),
```

Comment:
Again, I'm concerned about adding extra computation to the tx history endpoint (which needs to be really fast otherwise wallet sync is too slow)

Reply:
Well, this one in particular I was planning on removing it... I just put it here to be able to move forward with the api in paima, since I can't make cml work there. But I guess this could be optional with a flag on the request otherwise.